vochain/state.go

```diff
@@ -881,14 +903,18 @@ func (v *State) EnvelopeExists(processID, nullifier []byte, isQuery bool) (bool,
 }
 
 // iterateVotes iterates fn over state tree entries with the processID prefix.
-// if isQuery, the IAVL tree is used, otherwise the AVL tree is used.
+// if committed is true, the IAVL tree is used, otherwise the AVL tree is used.
+// TODO: does previous line is more descriptive then the followings 3 ?
```

Comment:
If you're not aware that `I` stands for Immutable then sadly not ... So the 3 following lines are better and maintain consistency with the other methods.

Reply:
And what does AVL stand for?

Reply:
https://pkg.go.dev/github.com/tendermint/iavl